source/MaterialXGenHw/Nodes/HwBitangentNode.h

```diff
@@ -3,15 +3,15 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#ifndef MATERIALX_HWBITANGENTNODE_H
-#define MATERIALX_HWBITANGENTNODE_H
+#ifndef MATERIALXGENHW_NODES_HWBITANGENTNODE_H
```

Comment:
This is an interesting idea for a future convention for our include guards, but for now I'd recommend keeping the existing convention, and we can consider global improvements to these naming conventions in a separate PR.